lib/knapsack_pro/formatters/time_tracker.rb

```diff
@@ -47,7 +47,8 @@ def example_finished(notification)
       def example_group_finished(notification)
         return unless top_level_group?(notification.group)
 
-        add_hooks_time(@group, @before_all, now - @time_all)
+        after_all = @time_all.nil? ? 0.0 : now - @time_all
```

Comment:
Fix to a bug when some calculation were done on `nil`. We need to set it to 0.0.

Reply:
It was happening when RSpec crashed. I simulated it with terminating the process. I was not able to invent a test for that so I did this fix https://github.com/KnapsackPro/knapsack_pro-ruby/commit/74785b1c8438a0d813f194ec4ea32bb7c5bb5dbe/lib/knapsack_pro/formatters/time_tracker.rb#diff--1 
I assumed it's better to have a safe defaults in TimeTracker just in case rather than nil.

I tried to reproduce the issue now and I tried to terminate the process at different moments but I was not able to reproduce it anymore. 

From what I recall, this bug in time tracker prevented me to show a summary of original error that happened so it was more confusing to users. That's why I fixed that.

Reply:
We can keep it. In the future, please keep notes (including a code comment) when we need something strange.

It's useful for reference and also to eliminate defensive code if it turns out later it's not needed anymore (because we cannot reproduce it).